From 24993b5a75c6565bc186541bf7cfc311894c0a85 Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@jumpnowtek.com>
Date: Tue, 13 Feb 2018 09:31:25 -0500
Subject: [PATCH 2/2] Revert "bcma: Fix 'allmodconfig' and BCMA builds on MIPS
 targets"

This reverts commit 664eadd6f44b3d71dcc62d0a825319000de0d5c9.

Unable to build bcma driver for lenovo s10e
---
 drivers/bcma/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/bcma/Kconfig b/drivers/bcma/Kconfig
index ba8acca..02d78f6 100644
--- a/drivers/bcma/Kconfig
+++ b/drivers/bcma/Kconfig
@@ -55,7 +55,7 @@ config BCMA_DRIVER_PCI
 
 config BCMA_DRIVER_PCI_HOSTMODE
 	bool "Driver for PCI core working in hostmode"
-	depends on MIPS && BCMA_DRIVER_PCI && PCI_DRIVERS_LEGACY
+	depends on MIPS && BCMA_DRIVER_PCI
 	help
 	  PCI core hostmode operation (external PCI bus).
 
-- 
2.7.4

